jsk_hand/softhand_ros/fc.rosinstall

```diff
@@ -0,0 +1,8 @@
+- git:
+    local-name: knorth55/softhand_ros
+    uri: https://github.com/knorth55/softhand_ros.git
+    version: master
+- git:
+    local-name: pazeshun/dynamixel_motor
```

Comment:
Please add list of PR included in forked repository.

Reply:
I'll work on this.

Reply:
Done.